src/display/GUI.java

```diff
@@ -0,0 +1,96 @@
+package display;
```

Comment:
Looks good but I'm not super familiar with GUI stuff so I'm just gonna assume it's been tested

Reply:
It has been tested and in the next PR, I would like to make it neater, like divide the GUI setup into helper methods with proper names to tell what they're doing.